iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -214,7 +219,7 @@ public void commitDropTable(org.apache.hadoop.hive.metastore.api.Table hmsTable,
   @Override
   public void preAlterTable(org.apache.hadoop.hive.metastore.api.Table hmsTable, EnvironmentContext context)
       throws MetaException {
-    super.preAlterTable(hmsTable, context);
+    setupAlterOperationType(hmsTable, context);
```

Comment:
We removed the `super.preAlterTable`. Is this intentional?

Reply:
Yes, this was intentional. The validation of supported alter operation types was moved from the superclass to the IcebergMetaHook (`setupAlterOperationType`)